airbyte-webapp/src/pages/SettingsPage/pages/ConnectorsPage/components/VersionCell.module.scss

```diff
@@ -18,18 +18,17 @@
   position: relative;
   background: colors.$white;
 
-  &::before {
+  &::after {
```

Comment:
Fixes a problem where this `(latest)` text was appearing above the DropdownMenu, because the z-index was too high